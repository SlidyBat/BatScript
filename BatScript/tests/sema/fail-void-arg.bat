def foo(void arg):
	return