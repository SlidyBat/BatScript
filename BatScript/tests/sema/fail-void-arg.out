[sema\fail-void-arg.bat:1:14] Error: 'void' is an invalid variable type
